c7n/policy.py

```diff
@@ -229,24 +229,31 @@ def run(self, *args, **kw):
             return
 
         with self.policy.ctx:
-            self.policy.log.info(
+            self.policy.log.debug(
                 "Running policy %s resource: %s region:%s c7n:%s",
                 self.policy.name, self.policy.resource_type,
                 self.policy.options.region or 'default',
                 version)
 
-            s = time.time()
-            resources = self.policy.resource_manager.resources()
-            rt = time.time() - s
+            region        = (self.policy.options.region or 'default').ljust(14, ' ')
```

Comment:
i'm a little wary of the hard coded alignments here, and also of mixing more presentation logic into the logs. 

afaics the root of the value chain here is just switching the info to debug on the double print.